DP-5636 Drop shadows on homepage - mobile

mariohernandez opened the pull request:
<!-- Please use TICKET Description of ticket as PR title (i.e. DP-1234 Add back-to link on Announcement template)  -->

## Description
https://jira.state.ma.us/browse/DP-5636
Some section breakers on homepage were showing box shadows that are not part
of the design requirements.  I went ahead and removed them.

## Related Issue / Ticket

- DP-5636
- N/A

## Steps to Test
* Visit Mass.gov's homepage
* Scroll through the page and look at the various row sections.  They should not have any box shadow.

## Screenshots
* Homepage showing box shadow: http://take.ms/KeuFT
* Homepage NOT showing box shadow: http://take.ms/BGHRj

## Additional Notes:

N/A

#### Impacted Areas in Application
Homepage mobile only.

#### @TODO
N/A

#### Today I learned...
Today I learned how to fix bugs in the mayflower repo.

legostud commented on styleguide/source/assets/scss/03-organisms/_stacked-row-section.scss:
```diff
@@ -1,7 +1,6 @@
 .ma__stacked-row-section {
 
   @media ($bp-large-max) {
-    box-shadow: 0 0.5rem 0.5rem -0.25rem rgba(1, 1, 1, 0.25);
```
This was added for use on the Guide Pages.  I doubt they've changed the design there.  

You will probably need to add a twig variable that adds a modifier class to the parent of the stacked row sections telling CSS to skip the box shadows.  `.ma__stacked-row-content .ma__stacked-row-content--no-shadows`

mariohernandez replied:
Thanks.  I'll work on that.

mariohernandez commented:
Updated PR by appending conditional of **is-front** to **.ma__stacked-row-section** to remove box-shadow from homepage.  Originally tried to add no-shadows conditional class but then realized **is-front** was already available on body.
Updated line ending settings from Windows to Unix to avoid invalid characters being introduced to commit.

mariohernandez commented:
Closed commit as these changes are being made within Drupal's them rather than mayflower.  A new PR will be created.
